test/routes.spec.js

```diff
@@ -1,279 +1,301 @@
-        done();
-      });
+    it('should not create a wave with missing data', (done) => {
+      chai
+        .request(server)
+        .post('/api/v1/waves')
+        .send({
+          WAVE_ID: '324521',
+          DAMAGE_ESTIMATE: '34235',
+        })
+        .end((error, response) => {
+          response.should.have.status(422);
+          response.body.error.should.equal("Expected format: { 'WAVE_ID': <String>, 'SOURCE_ID': <String>, 'YEAR': <String>, 'MONTH': <String>, 'LOCATION': <String>, 'MAXIMUM_HEIGHT': <String>, 'FATALITIES': <string>, 'FATALITY_ESTIMATE': <String>, 'ALL_DAMAGE_MILLIONS': <String>, 'DAMAGE_ESTIMATE': <String> }. You're missing a SOURCE_ID property.");
+          done();
+        });
     });
   });
 
   describe('DELETE /api/v1/sources/:id', () => {
-    it('should delete a source', done => {
-      chai.request(server)
-      .delete('/api/v1/sources/5586')
-      .end((error, response) => {
-        response.should.have.status(204);
-        done();
-      });
+    it('should delete a source', (done) => {
+      chai
+        .request(server)
+        .delete('/api/v1/sources/5586')
+        .end((error, response) => {
+          response.should.have.status(204);
+          done();
+        });
     });
   });
 
   describe('DELETE /api/v1/waves/:id', () => {
-    it('should delete a wave', done => {
-      chai.request(server)
-      .delete('/api/v1/waves/28689')
-      .end((error, response) => {
-        response.should.have.status(204);
-        done();
-      });
+    it('should delete a wave', (done) => {
+      chai
+        .request(server)
+        .delete('/api/v1/waves/28689')
+        .end((error, response) => {
+          response.should.have.status(204);
+          done();
+        });
     });
   });
 
   describe('PATCH /api/v1/sources', () => {
     const update = {
-      'SOURCE_ID': '5586',
-      'MAXIMUM_HEIGHT': '999',
-    }
+      LOCATION: 'DENVER',
+    };
 
-    it('should update source object', done => {
-      chai.request(server)
-      .patch('/api/v1/sources/5586')
-      .send(update)
-      .end((error, response) => {
-
-        done();
-      });
+    it('should update source object', (done) => {
+      chai
+        .request(server)
+        .patch('/api/v1/sources/5586')
+        .send(update)
+        .end((error, response) => {
+          console.log('response body', response.body.patchedSource);
+          response.should.have.status(200);
+          response.body.patchedSource.should.be.a('object');
+          response.body.patchedSource.should.have.property('LOCATION');
+          response.body.patchedSource.LOCATION.should.equal('DENVER');
```

Comment:
I just realized that I should also test to make sure that the other properties didn't change. 

I also need to add sad paths for this and the DELETE Tests